src/components/ebay-fake-tabs/README.md

```diff
@@ -0,0 +1,44 @@
+# ebay-fake-tabs
+
+## ebay-fake-tabs Usage
+
+```marko
+<ebay-fake-tabs>
+    <@heading>Tab 1</@heading>
+    <@heading>Tab 2</@heading>
+    <@heading>Tab 3</@heading>
+    <@panel>Panel 1</@panel>
```

Comment:
Fake tabs should only have one panel (as clicking a tab/heading loads a new page). In fact, might be best to use the word body or content or main instead of panel in order to get that point across.